File: _scratch/run/find_the_index_of_the_first_occurrence_in_a_string/sol.py
class Solution:
    def strStr(self, s, t):
        n, m = len(s), len(t)
        for i in range(n - m + 1):
            if s[i:i+m] == t:
                return i
        return -1
import sys
_l=sys.stdin.read().split('\n')
_T=int(_l[0]);_o=[]
for _i in range(_T):
    _s=_l[1+2*_i] if 1+2*_i<len(_l) else ""
    _t=_l[2+2*_i] if 2+2*_i<len(_l) else ""
    _o.append(str(Solution().strStr(_s,_t)))
print('\n'.join(_o))

File: _scratch/run/find_the_index_of_the_first_occurrence_in_a_string/sol.js
/**
 * @param {string} s
 * @param {string} t
 * @return {number}
 */
var strStr = function(s, t) {
    return s.indexOf(t);
};
const _l=require('fs').readFileSync(0,'utf8').split('\n');const _T=+_l[0];const _o=[];
for(let _i=0;_i<_T;_i++){const _s=(_l[1+2*_i]||'').replace(/\r$/,'');const _t=(_l[2+2*_i]||'').replace(/\r$/,'');_o.push(String(strStr(_s,_t)));}
console.log(_o.join('\n'));